pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -525,25 +525,87 @@ LogicalResult ConvertAtenOp<AtenTanhOp>::matchAndRewrite(
   }
 }
 
+// Convert a Aten::Relu to HLO
+// Relu(x) = 0 if x < 0 else x
+template <>
+LogicalResult ConvertAtenOp<AtenReluOp>::matchAndRewrite(
+    AtenReluOp op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  Location loc = op.getLoc();
+  Value input = adaptor.self();
+  auto inputTy = input.getType().cast<TensorType>();
```

Comment:
Is AtenReluOp always be ranked? maybe we need to check is the input type is RankedTensorType?

Reply:
done